src/transformers/models/autoformer/configuration_autoformer.py

```diff
@@ -162,13 +162,15 @@ def __init__(
         init_std: float = 0.02,
         use_cache=True,
         # Autoformer arguments
+        label_length: Optional[int] = None,
```

Comment:
This is great. I'm thinking after merging Autoformer, maybe it's possibe to add the non-autoregressive generation to Informer & vanilla also? @kashif

Reply:
Talked in Slack